desc/compute/_deprecated.py

```diff
@@ -0,0 +1,382 @@
+"""Deprecated compute functions.
+
+These are kept for verification purposes. They do not
+appear in the public documentation under the list of variables.
+"""
+
+from functools import partial
+
+from orthax.legendre import leggauss
+
+from desc.backend import imap, jit, jnp
+
+from ..integrals.bounce_integral import Bounce1D
+from ..integrals.quad_utils import (
+    automorphism_sin,
+    chebgauss2,
+    get_quadrature,
+    grad_automorphism_sin,
+)
+from ..utils import cross, dot, safediv
+from ._fast_ion import _cvdrift0, _drift1, _drift2, _v_tau
+from ._neoclassical import _bounce_doc, _dH, _dI
+from .data_index import register_compute_fun
+
+_bounce1D_doc = {
+    "num_well": _bounce_doc["num_well"],
+    "num_quad": _bounce_doc["num_quad"],
+    "num_pitch": _bounce_doc["num_pitch"],
+    "quad": _bounce_doc["quad"],
+}
+
+
+def _compute(fun, fun_data, data, grid, num_pitch, simp=False, reduce=True):
+    """Compute ``fun`` for each α and ρ value iteratively.
+
+    Parameters
+    ----------
+    fun : callable
+        Function to compute.
+    fun_data : dict[str, jnp.ndarray]
+        Data to provide to ``fun``. This dict will be modified.
+    data : dict[str, jnp.ndarray]
+        DESC data dict.
+    simp : bool
+        Whether to use an open Simpson rule instead of uniform weights.
+    reduce : bool
+        Whether to compute mean over α and expand to grid.
+        Default is true.
+
+    """
+    pitch_inv, pitch_inv_weight = Bounce1D.get_pitch_inv_quad(
+        grid.compress(data["min_tz |B|"]),
+        grid.compress(data["max_tz |B|"]),
+        num_pitch,
+        simp=simp,
+    )
+
+    def foreach_rho(x):
+        # using same λ values for every field line α on flux surface ρ
+        x["pitch_inv"] = pitch_inv
+        x["pitch_inv weight"] = pitch_inv_weight
+        return imap(fun, x)
+
+    for name in Bounce1D.required_names:
+        fun_data[name] = data[name]
+    for name in fun_data:
+        fun_data[name] = Bounce1D.reshape(grid, fun_data[name])
+    out = imap(foreach_rho, fun_data)
+    # Simple mean over α rather than integrating over α and dividing by 2π
+    # (i.e. f.T.dot(dα) / dα.sum()), because when the toroidal angle extends
+    # beyond one transit we need to weight all field lines uniformly, regardless
+    # of their spacing wrt α.
+    return grid.expand(out.mean(axis=0)) if reduce else out
+
+
+@register_compute_fun(
+    name="deprecated(effective ripple 3/2)",
```

Comment:
Is that the best name? Seems pretty long.
How about old effective ripple 3/2?

Reply:
I'll do it in a different PR

Reply:
issue for this?

Reply:
PR will be done soon so no issue